apps/web/src/pages/PoolCandidates/ViewPoolCandidatePage/components/ClaimVerification/ClaimRow.tsx

```diff
@@ -0,0 +1,116 @@
+import { useIntl } from "react-intl";
+import InformationCircleIcon from "@heroicons/react/20/solid/InformationCircleIcon";
+import CheckCircleIcon from "@heroicons/react/20/solid/CheckCircleIcon";
+import XCircleIcon from "@heroicons/react/20/solid/XCircleIcon";
+import { PropsWithoutRef, ReactNode, SVGProps } from "react";
+
+import {
+  ClaimVerificationResult,
+  Maybe,
+  Scalars,
+} from "@gc-digital-talent/graphql";
+import { Heading } from "@gc-digital-talent/ui";
+import {
+  formDateStringToDate,
+  formatDate,
+} from "@gc-digital-talent/date-helpers";
+import { commonMessages } from "@gc-digital-talent/i18n";
+
+interface VerificationMessageProps {
+  result: ClaimVerificationResult;
+  expiry?: Maybe<Scalars["DateTime"]["output"]>;
+}
+
+const VerificationMessage = ({ result, expiry }: VerificationMessageProps) => {
+  const intl = useIntl();
+  let message = intl.formatMessage({
+    defaultMessage: "This claim is unverified.",
+    id: "YiDdh6",
+    description: "Message when a claim as yet to be verified",
+  });
+
+  if (result === ClaimVerificationResult.Accepted) {
+    const expiryDate = expiry
+      ? formatDate({
+          date: formDateStringToDate(expiry),
+          formatString: "MMMM d, yyyy",
+          intl,
+        })
+      : intl.formatMessage(commonMessages.notProvided);
+
+    message = intl.formatMessage(
+      {
+        defaultMessage:
+          "This claim has been verified, expires on {expiryDate}.",
+        id: "1o5PUq",
+        description: "Message when a claim as been verified",
+      },
+      {
+        expiryDate,
```

Comment:
For priority, expiration will never be null. For veteran, it can be null. 
Does this null state copy look fine to you @Jerryescandon ? Sentence casing too
I did not see this state in the issue so I don't know if you had something specific in mind

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/5598e69c-1a76-443b-b0ac-5154e14771a8)

Reply:
@Jerryescandon says

> I would prefer if it was something like.
> "... , no expiration date was provided."

Reply:
Updated in [update null state message for expiration date](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10502/commits/adbbad2979c4a4c19ade3ef75bd203c2be5af2a7)